### PR TITLE
Upgrade ic js with latest governance

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Upgrade ic-js to get latest fields for 1-proposal.
Related to https://github.com/dfinity/ic-js/pull/389

# Changes

- upgrade ic-js to latest next
- set new flag to false

# Tests

Manually tested that the new field is visible.
image

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

File: CHANGELOG-Nns-Dapp.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ The NNS Dapp is released through proposals in the Network Nervous System. Theref
 * Faster formatting of shell and yaml files, by operating only on named or changed files.
 * Updated the calls to `docker-build` to use the `--network` flag.
 * Upgraded to Playwright 1.36.
+* Upgraded to ic-js 0.0.8-next-2023-07-24.1
```
Comment (peterpeterparker) [inline CHANGELOG-Nns-Dapp.md:57]: So far we never mentionned "bumping ic-js" in the changelog. Should we rather list why / what was bumped?

Like `* Update candid interface for NNS governance function xyz.`

Comment (mstrasinskis) [reply CHANGELOG-Nns-Dapp.md]: Good point, updated.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks

File: CHANGELOG-Nns-Dapp.md (hunk 1 above)
Comment (peterpeterparker) [reply CHANGELOG-Nns-Dapp.md]: Thanks

**Pull request merged**